modules/150-user-authn/hooks/generate_crowd_basic_auth_proxy_cert.go

```diff
@@ -349,19 +348,6 @@ func generateJob(registry, digest, csrb64 string) *batchv1.Job {
 	}
 }
 
-// ways are hardcoded because we know where these files exist on the Deckhouse clusters
-// if you want to grep client-ca-file from kube-apiserver command line, use `ps auxww` to avoid line clipping
-var (
-	script = `set -e
-ca_path="/etc/kubernetes/pki/front-proxy-ca.crt"
-key_path="/etc/kubernetes/pki/front-proxy-ca.key"
-
-csr_config='{"CN":"front-proxy-client","hosts":[""],"key":{"algo": "rsa","size": 2048},"signing":{"default":{"expiry":"72h","usages":["signing","key encipherment","requestheader-client"]}}}'
```

Comment:
Can't we just use the cfssl as a library for simplicity? To keep the initial code as much as possible.

For example, with cfssl the whole program looks like this:
```go
s, err := SignerFromConfig(&cli.Config{
	CAFile:    "/etc/kubernetes/pki/front-proxy-ca.crt",
	CAKeyFile: "/etc/kubernetes/pki/front-proxy-ca.key",
	CFG:       LoadConfig([]byte(`{"CN":"front-proxy-client","hosts":[""],"key":{"algo": "rsa","size": 2048},"signing":{"default":{"expiry":"72h","usages":["signing","key encipherment","requestheader-client"]}}}`)),
})
if err != nil {
	return
}

cert, err := s.Sign(signer.SignRequest{Request: os.Getenv("CSR")})
if err != nil {
	return
}
```

Reply:
Fixed. Changed everything to use cfssl module.